src/pybind/mgr/tox.ini

```diff
@@ -47,6 +47,7 @@ commands =
         insights/ \
         pg_autoscaler/ \
         progress/ \
+        cli_api/ \
```

Comment:
You also have to add `cli_api` to the other checks (flake8, mypy, ...) and fix the errors:
```
diff --git a/src/pybind/mgr/tox.ini b/src/pybind/mgr/tox.ini
index 301bc9569de..7d58b6e9d4b 100644
--- a/src/pybind/mgr/tox.ini
+++ b/src/pybind/mgr/tox.ini
@@ -42,12 +42,12 @@ commands =
         mgr_util.py \
         tests/ \
         cephadm/ \
+        cli_api/ \
         mds_autoscaler/ \
         orchestrator/ \
         insights/ \
         pg_autoscaler/ \
         progress/ \
-        cli_api/ \
         snap_schedule}
 
 [testenv:mypy]
@@ -65,6 +65,7 @@ commands =
            -m alerts \
            -m balancer \
            -m cephadm \
+           -m cli_api \
            -m crash \
            -m dashboard \
            -m devicehealth \
@@ -105,6 +106,7 @@ modules =
     alerts
     balancer
     cephadm
+    cli_api
     crash
     devicehealth
     diskprediction_local
@@ -127,6 +129,7 @@ modules =
     alerts
     balancer
     cephadm
+    cli_api
     crash
     devicehealth
     diskprediction_local
```

Errors found for mypy (also check flake8):
```
cli_api/__init__.py:7: note: In module imported here:
cli_api/module.py: note: In member "__init__" of class "ThreadedBenchmarkRunner":
cli_api/module.py:53: error: Need type annotation for 'jobs'
cli_api/module.py: note: In member "timer" of class "ThreadedBenchmarkRunner":
cli_api/module.py:91: error: Incompatible types in assignment (expression has type "float", variable has type "int")
```